### PR TITLE
first update for 2019

### PR DESCRIPTION
**Pull request by braverock**
- Add new committee members for 2019
- comment out 2018 program div
- uncomment CFP
- Update dates for 2019
- copy 2018 information to RinFinance2018 folder
- update CFP for 2019, including links to UIC code of conduct, etc.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -649,11 +660,14 @@ <h1 class="mdl-mega-footer--heading">Previous Conferences</h1>
               <h1 class="mdl-mega-footer--heading">Committee</h1>
               <ul class="mdl-mega-footer--link-list">
                 <li><a href="http://gib.people.uic.edu/">Gib Bassett</a></li>
-                <li>Peter Carl</li>
+                <li><a href="https://www.linkedin.com/in/pbakosova/">Petra Bakosova</a></li>
```
Comment (eddelbuettel) [inline docs/index.html:663]: Please ask if that is the URL she wants listed.

File: docs/index.html [hunk 2]
```diff
@@ -649,11 +660,14 @@ <h1 class="mdl-mega-footer--heading">Previous Conferences</h1>
...
+                <li><a href="https://www.linkedin.com/in/peter-carl-59160/">Peter Carl</li>
```
Comment (eddelbuettel) [inline docs/index.html:664]: Ditto

File: docs/index.html [hunk 3]
```diff
@@ -649,11 +660,14 @@ <h1 class="mdl-mega-footer--heading">Previous Conferences</h1>
...
                 <li><a href="http://dirk.eddelbuettel.com">Dirk Eddelbuettel</a></li>
                 <li><a href="http://braverock.com/brian/">Brian Peterson</a></li>
+                <li><a href="https://www.linkedin.com/in/soumyakalra/">Soumya Kalra</a></li>
```
Comment (eddelbuettel) [inline docs/index.html:667]: Ditto

File: docs/index.html [hunk 4]
```diff
@@ -649,11 +660,14 @@ <h1 class="mdl-mega-footer--heading">Previous Conferences</h1>
...
                 <li><a href="http://daler.people.uic.edu">Dale Rosenthal</a></li>
                 <li><a href="http://www.lemnica.com">Jeffrey Ryan</a></li>
+                <li><a href="http://rinfinance.com">Justin Shae</a></li>
```
Comment (eddelbuettel) [inline docs/index.html:670]: Looks wrong.

**Review by eddelbuettel — COMMENTED** [review]
Let's wait til I have the main file moved and then make changes.

**Comment by eddelbuettel** [comment]
I am just going to close this one as it has several bulk changes meddled together several of which need discussion.  

So just say "no" to waterfall PRs.  Individual issues can be discussed.

File: docs/index.html (hunk 1 above)
Comment (braverock) [reply docs/index.html:663]: I defaulted to LinkedIn profiles where I had them, but I have also asked Petra, Soumya, and Justin what link they would prefer.

Comment (eddelbuettel) [reply docs/index.html:663]: We do have a mailing list so that these things are not clandestine and can be validated. I would use it.

**Comment by braverock** [comment]
moving one file and making changes to another file hardly count as a waterfall PR.  I see no reason to break this into multiple commits.

File: docs/index.html (hunk 1 above)
Comment (braverock) [reply docs/index.html:663]: Emailed the list for discussion.

**Pull request closed without merging**